File: teoria01/hello.c
#include <stdio.h>
int main() {
    printf("Boa noite a todos!\n");

    for(int i = 1; i<=3; i++) {
        printf("Notas do aluno %d\n", i);
        int n;
        n = 2;
        float p1;
        printf("nota da p1: ");
        scanf("%f", &p1);
        printf("nota da p2: ");
        float p2;
        scanf("%f",&p2);
        float media;
        media = (p1 + p2)/n;
        
        printf("SUA MEDIA É: %f\n", media);
        if (media >= 6){
            printf("Foi aprovado\n");
        }
        else {
            printf("Foi reprovado\n");
        }
    }
    return 0;
}

File: teoria01/hello.py
print('Boa Noite!')
for i in range(1,4):
    print("\n Notas do aluno:", i)
    n=2 
    p1= float(input("nota da p1: "))
    p2= float(input("digite a nota da p2: "))
    media=(p1+p2)/n
    print(f'Sua média é {media}')
    if media >= 6:
        print("Aprovado")
    else:
        print("Recuperação")
        
